users/models.py

```diff
@@ -291,6 +291,20 @@ def github_profile_url(self):
             return None
         return f"https://github.com/{self.github_username}"
 
+    @staticmethod
+    def get_user_by_github_url(url: str):
+        if not url:
+            return None
+        github_user = url.split("/")[-1]
```

Comment:
Do we need to guard against a trailing slash?

```suggestion
        github_user = url.rstrip("/").split("/")[-1]
```